README.md

```diff
@@ -64,6 +64,7 @@ helps for trackability, recollection, debugging, and fun! :neckbeard:
 - :envelope: `:envelope:` email templates or content
 - :snowflake: `:snowflake:` i18n/l10n, partnerification, white labeling
 - :hammer: `:hammer:` administration, moderation
+- :cowboy_hat_face: `:cowboy_hat_face:` unreviewed code
```

Comment:
I would also add something like `cowboy  commit` or `cowboy code`. Because semantically, that's what this is intended for :wink:

Reply:
👍